var/spack/repos/builtin/packages/dos2unix/package.py

```diff
@@ -14,7 +14,7 @@ class Dos2unix(MakefilePackage):
 
     version('7.3.4', sha256='8ccda7bbc5a2f903dafd95900abb5bf5e77a769b572ef25150fde4056c5f30c5')
 
-    depends_on('gettext', type='build')
+    depends_on('gettext')
```

Comment:
Why not simply add 'link' to the types?

Reply:
The default is `type=('build', 'link')`. Do you want me to explicitly add this? We haven't enforced this in the past, although it might be a good idea.

Reply:
Good point.  I like explicit but will leave that up to you.  Just let me know.

Reply:
I think it's fine as is unless we change our policy.